diff --git a/bin/guardctrl b/bin/guardctrl
index 2514015..e2e9c24 100755
--- a/bin/guardctrl
+++ b/bin/guardctrl
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec python3 -u -m guardctrl "$@"
+exec /opt/anaconda1anaconda2anaconda3/bin/python -u -m guardctrl "$@"
diff --git a/bin/guardian b/bin/guardian
index ff80123..c9f1b89 100755
--- a/bin/guardian
+++ b/bin/guardian
@@ -1,3 +1,3 @@
 #!/bin/sh
 # FIXME: should we add other command line options?  -B, -s?
-exec python3 -u -m guardian "$@"
+exec /opt/anaconda1anaconda2anaconda3/bin/python -u -m guardian "$@"
diff --git a/bin/guardlog b/bin/guardlog
index d304320..9dcf934 100755
--- a/bin/guardlog
+++ b/bin/guardlog
@@ -20,5 +20,5 @@ exec xterm \
     -fa Monospace \
     -fs 10 \
     -title "guardctrl: $*" \
-    -e "python3 -u -m guardctrl log -n 1000 -f $*" \
+    -e "/opt/anaconda1anaconda2anaconda3/bin/python -u -m guardctrl log -n 1000 -f $*" \
     &
diff --git a/bin/guardmedm b/bin/guardmedm
index 931612a..c5d7fe9 100755
--- a/bin/guardmedm
+++ b/bin/guardmedm
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec python3 -m guardian.medm "$@"
+exec /opt/anaconda1anaconda2anaconda3/bin/python -m guardian.medm "$@"
diff --git a/bin/guardutil b/bin/guardutil
index f5f1a17..46ba542 100755
--- a/bin/guardutil
+++ b/bin/guardutil
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec python3 -u -m guardian.guardutil "$@"
+exec /opt/anaconda1anaconda2anaconda3/bin/python -u -m guardian.guardutil "$@"
